### PR TITLE
Admin command line supports list server command

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Support to list server with kyuubi-admin/kyuubi rest client.

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- UT for AdminRestAPI
- UT for AminCtlArgument
- UT for AdminCtl

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by turboFei** [comment]
cc @zwangsheng

**Comment by turboFei** [comment]
tests passed @pan3793

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/util/Render.scala [hunk 1]
```diff
@@ -45,6 +45,19 @@ private[ctl] object Render {
...
   }
 
+  def renderServerNodesInfo(serverNodesInfo: Seq[ServerData]): String = {
+    val title = s"Server Node List (total ${serverNodesInfo.size})"
+    val header = Array("Namespace", "Instance", "Attributes", "Status")
+    val rows = serverNodesInfo.map { server =>
+      Array(
+        server.getNamespace,
+        server.getInstance,
+        server.getAttributes.asScala.map(kv => kv._1 + "=" + kv._2).mkString("\n"),
```
Comment (pan3793) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/util/Render.scala:55]:
```suggestion
        server.getAttributes.asScala.map { case (k, v) => s"$k=$v").mkString("\n"),
```

**Review by pan3793 — APPROVED** [review]

**Comment by turboFei** [comment]
merged to master, thanks

**Pull request closed without merging**